robottelo/ui/locators/base.py

```diff
@@ -1687,8 +1687,9 @@
         By.XPATH, "//td/a[contains(., '%s')]/following::td/div/"
                   "button[contains(@class, 'dropdown-toggle')]"),
     "contentviews.remove_ver": (
-        By.XPATH, ("//td/a[contains(., '%s')]/following::td//"
-                   "a[contains(@ng-click, 'version-deletion')]")),
+        By.XPATH,
+        ("//td/a[contains(., '%s')]/following::td[@class='col-sm-2'][1]"
```

Comment:
For example you have 'Version 1.0', 'Version 2.0' and 'Version 3.0'
In Google Chrome console how many elements are returned for `//td/a[contains(., 'Version 3.0')]/following::td[@class='col-sm-2']//a[contains(@ng-click, 'version-deletion')]`?
Not one, right? :)